### DISCUSSION TITLE
My final commit

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#664] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
It was a nice run helping you all out in your little rampant craziness, and as a final tribute I have added what seems to be the kryptonite of extensions. Works on any extension or app, needs no perms whatsoever, and is almost impossible to patch. You'll see it soon if echo decides to add it.

**Comment by srets** [comment]
its basically the same thing as ltmeat
